src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,45 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+        // initialize field values using setters or constructor
+        book.setTitle("Java");
+        book.setPrice(BigDecimal.valueOf(100));
+        Book createdBook = bookDao.create(book);
+        System.out.println("Book created: " + createdBook);
+        System.out.println("-----------------------------------------------------");
+        // test other methods from BookDao
+        Optional<Book> optionalBook1 = bookDao.findById(5L);
```

Comment:
Your code return 
book id = 5: Optional.empty
book id = 100: Optional.empty

Let's use bookId that was created